library/core/src/intrinsics/mod.rs

```diff
@@ -4457,7 +4465,11 @@ pub const unsafe fn copy_nonoverlapping<T>(src: *const T, dst: *mut T, count: us
 /// ```
 #[doc(alias = "memmove")]
 #[stable(feature = "rust1", since = "1.0.0")]
-#[rustc_allowed_through_unstable_modules]
+#[cfg_attr(bootstrap, rustc_allowed_through_unstable_modules)]
+#[cfg_attr(
+    not(bootstrap),
+    rustc_allowed_through_unstable_modules = "import this function via `std::mem` instead"
+)]
```

Comment:
```suggestion
#[cfg_attr(
    not(bootstrap),
    rustc_allowed_through_unstable_modules(deprecated = "import this function via `std::mem` instead")
)]
```

Reply:
I have no idea how to implement that with reasonable effort, sorry.